src/renderer/components/app.tsx

```diff
@@ -176,7 +178,7 @@ export class App extends React.Component {
     return (
       <Router history={history}>
         <ErrorBoundary>
-          <MainLayout>
+          <MainLayout sidebar={<Sidebar/>} footer={<Dock/>}>
```

Comment:
Why not to render `<SIdebar/>` directly in `MainLayout`? Are the some use-cases when we need to hide or replace it?

Reply:
Catalog has its own sidebar. Consider this